core/shared/src/main/scala/zio/Exit.scala

```diff
@@ -193,10 +193,10 @@ sealed trait Exit[+E, +A] extends Product with Serializable { self =>
 
 object Exit extends Serializable {
 
-  final case class Success[A](value: A)        extends Exit[Nothing, A]
-  final case class Failure[E](cause: Cause[E]) extends Exit[E, Nothing]
+  final case class Success[A](value: A)            extends Exit[Nothing, A]
+  final case class Failure[E](cause: zio.Cause[E]) extends Exit[E, Nothing]
```

Comment:
@iravid I was not able to avoid mentioning the full package name here(since the object has a similar value with the same name).
Would love some help here.

Reply:
yeah that's probably unavoidable (unless you import and rename the top-level cause...)